### DISCUSSION TITLE
can prisma generate typescript file with my comment?

### DISCUSSION THREAD

**Discussion by xiaoxintang** [prisma/prisma#25396] [Q&A]
### Question

prisma file
```prisma
model User {
  id        Int      @id @default(autoincrement())
  /// 邮箱
  email     String?  @unique
}
```

i want auto generate ts file  look's like

```ts
export type UserWhereInput = {
    AND?: UserWhereInput | UserWhereInput[]
    OR?: UserWhereInput[]
    NOT?: UserWhereInput | UserWhereInput[]
    id?: IntFilter<"User"> | number
    /**邮箱*/
    email?: StringNullableFilter<"User"> | string | null
   
  } 
```

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @xiaoxintang 👋 

This is currently not possible. However, we have an open [feature request](https://github.com/prisma/prisma/issues/5113) for this use case. Please leave a 👍 and document your use case to help our engineers prioritize.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
